railties/lib/rails/generators/rails/scaffold_controller/templates/controller.rb

```diff
@@ -49,7 +77,27 @@ def destroy
     @<%= orm_instance.destroy %>
     redirect_to <%= index_helper %>_url, notice: <%= "'#{human_name} was successfully destroyed.'" %>
   end
+  <% if collection_routing? %>
+  # Collection routes
+  # PATCH <%= route_url %>/1..10
+  def update_many
+    <%= orm_class.find(class_name, "#{singular_table_name}_ids") %>.each do |<%= singular_table_name %>|
+      <%= singular_table_name %>.update(params[:<%= plural_table_name %>][:"#{<%= singular_table_name %>.id}"])
+    end
+    redirect_to @<%= singular_table_name %>, notice: <%= "'#{human_name}s were successfully updated.'" %>
+  end
 
+  # PUT <%= route_url %>/1..10
+  def replace
+    <%= orm_class.destroy_all(class_name) %>
+    create
+  end
```

Comment:
> More importantly this completes the HTTP verbs for our case.

I don't think that's a valid justification for adding a method if it does not solve any actual problem.

If this method is added, then it should have documentation (with examples) and guides. 
If we can't come up with good examples then developers will end up confused, rather than happy.

Given this method has already confused me and has caused you troubles in finding the right name, it might be worth to drop this method, and focus on the ones that have a clear value.